test-api-locally.md

```diff
@@ -48,24 +58,52 @@ You will need to reload the file to take effect using the following command and
 % source ~/.zshrc
 ```
 
-### 3. Update Properties files
+## 3. Update Properties files
 
 Before you can run a test against the new database you will need to tell Galasa where your database is. To do this insert the below lines into the corresponding files in your .galasa direcotry.
 
-Please note that if you make future runs against an ecosystem you will need to override these properties.
+Please note that if you make future runs against a galasa ecosystem you will need to override these properties.
 
 #### cps.properties
 
-```
+```shell
```

Comment:
Need to mention that this will need to be created on the initial start-up in etcd as well in order for the CouchDB instance to work